extra/1-factorial.js

```diff
@@ -9,8 +9,19 @@
 */
 
 function factorial(input) {
-  // TODO
-}
+  let lastNumber = 1;
+    for (i=input; i >= 1; i--){
+      console.log("before", lastNumber)
```

Comment:
I like the way you're using `console.log` to see what's happening when your code is running 😄